todos.json

```diff
@@ -0,0 +1,3 @@
+{
+  "name": "Marga"
+}
```

Comment:
This seems like a change for the next PR :)

Reply:
yes.. I am still totally aware of this file which was part of my answer finding process :oD hihi